lib/src/components/NxToast/NxToast.scss

```
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-toast__wrapper {
+  // nx-toast__wrapper has position absolute to ensure toasts stay in the top right corner of the page.
+  // Using display:flex and flex-end ensures the right side of the toast is aligned with the right side of 
+  // nx-page-main. The nx-toast_container has position fixed applied so it remains fixed while scrolling.
+  position: absolute;
+  top: 0;
+  right: 0;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: var(--nx-spacing-6x);
+}
+
+.nx-toast__container {
+  position: fixed;
+  width: 500px;
+}
+
+.nx-toast {
+  position: relative;
+  animation: animationIn 0.4s linear;
+  right: 0;
+  opacity: 1;
+
+  &.nx-toast--closing {
+    animation-name: animationOut;
+  }
+
+  .nx-alert {
+    margin: var(--nx-spacing-2x) 0 0 0;
+  }
+}
+
+//The 74px in each animation refers to the total height of each toast: the content itself,
+//plus the padding and margin-top applied. Setting the max-height of each toast to 74px within
+//the animation allows for each subsequent toast to smoothly move up or down as toasts are
+//added and removed.
+@keyframes animationIn {
+  0% {
+    margin-top: 0;
+    max-height: 0px;
+    right: -100%;
+    opacity: 0;
+  }
+  40% {
+    margin-top: var(--nx-spacing-2x);
+    right: -100%;
+    opacity: 0;
+    max-height: 74px;
+  }
+}
+
+@keyframes animationOut {
+  75% {
+    max-height: 74px;
+    right: -100%;
+    opacity: 0;
+  }
+  100% {
+    max-height: 0px;
+    right: -100%;
+    opacity: 0;
+  }
+}
+
+/*
+  If the user has expressed their preference for reduced motion,
+  then only animate the fade effect.
+*/
+@media (prefers-reduced-motion: reduce) {
+  .nx-toast {
+    animation: reducedAnimationIn 0.4s linear;
+
+    &.nx-toast--closing {
+      animation: reducedAnimationOut 0.4s linear;
```

Since all of our animations use the same duration and easing, this can be simplified a little
```suggestion
    animation-name: reducedAnimationIn;

    &.nx-toast--closing {
      animation-name: reducedAnimationOut;
```